inf-terraform-aws/files/lib/scripts/aws/check_conf.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+#
+# Author: Erhard Wais
+#         erhard.wais@boehringer-ingelheim.com
+#
+# This script does some basic checks on the AWS QS and reports potential issues.
+# will be triggered via "make check-config"
+
+# TODO:
+# - Return error in case of missconfig
+
+set -e
+set -o pipefail
+
+#CONST
+SHAREDBUCKET="bitfstate01"
+EDPUSER="edpawsqs"
+EDPPOLICY="BI-EDPAWSQS-Policy"
+DEFAULTBUCKET="<your_bucket_name>"
+DEFAULTACCOUNT="<your_aws_account_id>"
+DOTS="........................................................................."
+GITROOT=`git rev-parse --show-toplevel`
+
+BUCKET=
+ACCOUNT=
+MESSAGE=
+HASAWSCONFIGURED=0
+
+# functions
+function format_message() {
+  MESSAGE=$1
+  local offset=${#MESSAGE}
+  MESSAGE="$MESSAGE${DOTS:offset:((${#DOTS} - offset))}"
+}
+
+function ok() {
+  format_message "$1"
+  echo -e "$MESSAGE\033[42mPassed\033[0m"
+}
+function nok() {
+  format_message "$1"
+  echo -e "$MESSAGE\033[41mFailed\033[0m"
+}
+function warn() {
+  format_message "$1"
+  echo -e "$MESSAGE\033[44m Warn \033[0m"
+}
+function note() {
+  format_message "$1"
+  echo -e "$MESSAGE"
+}
+
+function check_backend() {
+  BUCKET=`grep "bucket =" backend.tf | awk -F '=' '{print $2}'|tr -d '"'|xargs`
+  case $BUCKET in
+    $DEFAULTBUCKET) nok "TF Backend is not configured. Check your backend.tf file";;
+    $SHAREDBUCKET) ok "TF Backend configured to \"$BUCKET\"";;
+    "") nok "There is no backend specified. Update your backend.tf file";;
+    *) warn "You don't use the shared Backend. Backend is set to \"$BUCKET\"";;
```

Comment:
(shared doesn't matter it could also be dedicated).

TF Backend is not configured, update backend.tf file.